roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -0,0 +1,49 @@
+# These tasks are run when aci_mode is set
+
+- name: check aci-gw container image
+  shell: docker inspect contiv/aci-gw
+  register: docker_aci_inspect_result
+  ignore_errors: yes
```

Comment:
sure, I think `contiv-compose` installation can move to `docker` role as it is just a modified version of `docker-compose`. That will make this entire file aci specific and then we can move `run_once` out.